packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -445,6 +463,21 @@ impl EstimablePredicates for CreateTransaction {
     }
 }
 
+impl ValidablePredicates for CreateTransaction {
```

Comment:
Because the implementations for `CreateTransaction` and `ScriptTransaction` are the same, I would suggest we put it in the macro `impl_tx_wrapper` like so:
```rust
        impl ValidablePredicates for $wrapper {
            fn validate_predicates(
                self,
                consensus_parameters: &ConsensusParameters,
                block_height: u32,
            ) -> Result<()> {
                let checked = self
                    .tx
                    .into_checked(block_height.into(), consensus_parameters)?;
                let check_predicates_parameters: CheckPredicateParams = consensus_parameters.into();
                checked.check_predicates(&check_predicates_parameters)?;

                Ok(())
            }
        }
```